stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -86,9 +80,13 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
     check_consistent_sizes(function, "Vector of intercepts", alpha,
                            "Vector of dependent variables", y);
 
+  if (size_zero(y, x, beta))
+    return 0.0;
+
   if (!include_summand<propto, T_x, T_alpha, T_beta>::value)
     return 0.0;
 
+  T_partials_return logp(0.0);
```

Comment:
This is great you moved this up.  All these `0.0` can be `0`, but that's mostly cosmetic because the compiler will deal with the type.

Reply:
Changed all the 0.0 to 0 in the touched files.